### PR TITLE
Add method signature to invalid number of solidity args error

### PR DESCRIPTION
**Pull request by angus-hamill**
It can be tough as a developer to determine the source of this error when executing a number of transactions (eg. a truffle test)

It may also be worth including the arguments themselves?

### CODE REVIEW

**Comment by joshstevens19** [comment]
Yeah can't see how this can hurt, if your going to do it for `InvalidNumberOfSolidityArgs` you may as well do the same for `InvalidNumberOfRPCParams` as that would be pretty useful as well :)

Why don't you do the same for that and i will then relook. 

Thanks for the PR 👍

**Comment by joshstevens19** [comment]
Looks good to me 👍 We need 2 approvals to get this in @nivida can you approve and merge when you get a second 

Thanks again @angus-hamill

**Review by joshstevens19 — APPROVED** [review]

**Review by nivida — APPROVED** [review]

**Review by yvettep321 — COMMENTED** [review]
thank you

**Pull request merged**
